include/dsn/dist/replication/replication_ddl_client.h

```diff
@@ -274,10 +278,31 @@ class replication_ddl_client
         }
     }
 
+    /// Send request to single replica server synchronously.
+    template <typename TRpcHolder, typename TResponse = typename TRpcHolder::response_type>
+    error_code call_rpc_async(dsn::rpc_address address,
```

Comment:
It's my mistake: https://github.com/XiaoMi/rdsn/blob/1755c53e455443815b4fe5940f95a1f0994b3f1c/include/dsn/dist/replication/replication_ddl_client.h#L247-L250

They should be named `XXXX_sync`, you can help me modify it